### ISSUE TITLE
Votrax (and more?) sound issues for GTS80

### ISSUE THREAD

**Issue by toxieainc** [vpinball/pinmame#63]
From https://www.vpforums.org/index.php?showtopic=27063&page=73#entry491401 (and half a dozen of the following posts) :
Just noticed that Gottlieb system 80 still has sound issues since many versions prior to 3.4 beta.

Please compare the sound produced by the slingshots of 'The Amazing Spider-Man' with version 2.5 and 3.3 or 3.4beta. It's only correct with the 2.5 version.
There are also sound issues for 'Hauted Hause', 'Black Hole', ...

---

I don't know what as been changed in the code at this time so all version after this one have a weird distortion and lose some sound effects.

---

Amazing Spiderman (and maybe system 80 in general?) seems to need a pitch correction

---

About the spiderman. Taking this completely out of my memory; and as I'm getting older ... etc. Isn't there some kind of knob you can turn for the pitch ? Haven't tried the latest vpm yet, only lies there ready to be installed. What I've noticed though is that setup.exe crashed on Win7 64bit without excluding the compression. It is mentioned earlier in this thread and I had to go back to the release that jsm gave me to get it to work. It is also not mentioned in the release that  the users should do a uninstall/install - or - have I missed that part ? Since I haven't tried the very latest yet, I would not know. But, I expect that blackhole is still giving me issues. What I've seen is that playing that game once; a reboot afterwards is highly recommended.

---

Yes on black hole, if you play once, Votrax will be a bit fubar on the next game
The rest of the sound will be ok, but Votrax will be broken up degrading into just staticy noise.
Restart the ROM and it will be fine again

In spiderman, while the sounds sound correct as far as construction and speed played, the pitch is definitely way off much too low
I somewhat changed the sound by turning the CPU up to 200%
not exactly perfect though

---

I don't think you should change the default for Spiderman. Here, default plays quite close to what I remember; which isn't a proof on it's own ;) The one on youtube, to me sounds too high pitched.

---

You probably did not mean to
But you got a slight bit better on the votrax reset after game over on black hole

It still breaks kind of, but you can now hear him, though kind of faint.

The sound still breaks up towards the tail end of that game over explosion sound, and then the votrax goes scratchy
but now his voice is still audible instead of just total static

**Comment by toxieainc** [comment]
Regarding Black Hole:
From https://www.vpforums.org/index.php?showtopic=27063&page=65#entry473855 :
Want to know something bizarre?

vpinmame64 appears to do a bit better job of emulating the votrax in blackhole than 32bit

not perfect, but it is not going totally unintelligible after the 1st game.

not sure why, maybe something as silly as simply different memory addresses/maps simply dodging the issue partially

---

And below: https://www.vpforums.org/index.php?showtopic=27063&page=65#entry474345 :
Seems our old friend Black Hole is still a bastard. First of all, this test is made with CP's 1.1 version, and I'm writing this here as I can't see it being anything by pinmame. So, the issue that I'm seeing is that after a boot, first time you play the game/load the rom it is just fine. If you exit and try again. You have a very high chance of needing to reboot. I say high chance because from my 7-8 rebooting I was able to play it the second time around twice, before the emulation got sick.

Sick in that sense that the flippers where delayed, sometimes my very much. I saw once that the ball didn't even get ejected. Sometime the flippers could feel fast and nice, but would all of a sudden manage to not even notice that I've released the flippers and stayed up/enabled. Pressing fast on the flipper, the could notice it and fall back, but, not always. Only way to fix it was reboot.

During my testing. I tried different versions of VP - 1.6.2 - 10.7.127 and a compile I did yesterday changing some constants for the nudge filter. Different version used as the "first one" after clean boot. No difference. After 6 reboots. Upgrade to pinmame created 3 days ago, currently latest "sc" 32bit. Setup via uninstall/install. No change. Reboot, and it would still working perfectly, first time around. Last time I was able to start the emulation twice before it got "sick". Decided to fire up a couple of other sys 80. Panthera and Incredible Hulk, when I was still in this Black Hole being sick situation, restarted those 3 times each, no issued with flippers.

So, to me it seems like a isolated problem to the title. What I have NOT tested yet. Is a different rom for that title. Only tried with blackhole7.
